src/templates/form-page.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+import Grid from '@material-ui/core/Grid';
+import Layout from '../components/Layout';
+import Content, { HTMLContent } from '../components/Content';
+import FeatureList from '../components/FeatureList';
+
+export const FormPageTemplate = ({
+  title,
+  content,
+  contentComponent,
+  features,
+}) => {
+  const PageContent = contentComponent || Content;
+
+  return (
+    <div className="siteContent">
+      <div className="siteContent-inner">
+        <Grid container spacing={5} alignItems="center">
+          <Grid item md={7} xs={12}>
+            <h2>{title}</h2>
+            <PageContent className="content" content={content} />
+          </Grid>
+          <Grid item md={5} xs={12}>
+            <FeatureList items={features} />
+          </Grid>
+        </Grid>
+      </div>
+    </div>
+  );
+};
+
+FormPageTemplate.propTypes = {
+  title: PropTypes.string.isRequired,
+  content: PropTypes.string,
+  contentComponent: PropTypes.func,
+  features: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Do we need to add any type of existing check to hide/show these elements in the render if optional props do not exist?

example: If `content` is undefined, buy `contentComponent` is provided, will `undefined` get passed into the content prop?

Reply:
I was thinking that too, and decided against adding it, just cause the PR was getting big, and it's not strictly in the design (figured I'd do a follow up PR without an issue number). I have the code to collapse the header without content, and the feature list if that list is empty in an experimental branch I can clean up and throw in later tonight.

Reply:
New commit adds conditional render for optionals. Screenshots in the description 😄